test/Sema/explicit_lifetime_dependence_specifiers.swift

```diff
@@ -22,7 +22,7 @@ struct WrapperStruct {
   let k: Klass
 }
 
-func invalidLifetimeDependenceOnEscapable(_ w: borrowing WrapperStruct) -> Klass {
+func invalidLifetimeDependenceOnEscapableResult(_ w: borrowing WrapperStruct) -> _borrow(w) Klass { // expected-error{{lifetime dependence specifiers can only be specified on ~Escapable results}}
```

Comment:
probably don't need to say 'specifiers' twice. "dependence can only be specified on"